_data/toc.yaml

```diff
@@ -15,6 +15,9 @@ horizontalnav:
 - title: Samples
   path: /samples/
   node: samples
+- title: Contribute
+  path: /contribute/overview/
+  node: contribute
   # TODO unify navbbar links: homepage currently has a custom "articles" link
   hide_home: true
```

Comment:
I don't think we should hide the contribute section from the landing page. But, it looks like that would also require some CSS changes, too. The home page already contains the additional "Articles" link, and adding contribute as well makes it rather wide. The media query looks like it's set to collapse the nav menu at 768px, which is narrower than the width of the navbar if we also add contribute.

Easy solution would be to remove the Articles (blog) link. Do we really need it? :D